protocol/avdl/keybase1/notify_ctl.avdl

```diff
@@ -26,6 +26,8 @@ protocol notifyCtl {
     boolean deviceclone;
     boolean chatattachments;
     boolean wallet;
+    boolean unverifiedteamlist;
```

Comment:
Don't think you need these, they can just show up if you choose `team`

Reply:
👍